source/includes/extracts-compatibility.yaml

```diff
@@ -4,6 +4,7 @@ content: |
    {+dbtools+} version ``{+release+}`` supports the following versions
    of the MongoDB server:
 
+   - MongoDB 8.0
    - MongoDB 7.0
    - MongoDB 6.0
    - MongoDB 5.0
```

Comment:
[non-blocking question] Should we remove v4.2 and v4.4 since they're EOL'd?

Reply:
Thanks @ajhuh-mdb - good catch. I'll bring it up with the tech reviewer and make the change if they give the okay.

Reply:
We don't need to remove them because the DB tools still support v4.2 and v4.4 at the moment.